src/v2/style-guide/index.md

```diff
@@ -45,18 +45,18 @@ Some features of Vue exist to accommodate rare edge cases or smoother migrations
 
 
 
-## Priority A Rules: Essential (Error Prevention)
+## 優先度 A ルール: 不可欠 (エラー防止)
 
 
 
-### Multi-word component names <sup data-p="a">essential</sup>
+### 複数単語コンポーネント名 <sup data-p="a">不可欠</sup>
 
-**Component names should always be multi-word, except for root `App` components.**
+**ルートの `App` コンポーネントを除き、コンポーネント名は常に複数単語であるべきです。**
 
-This [prevents conflicts](http://w3c.github.io/webcomponents/spec/custom/#valid-custom-element-name) with existing and future HTML elements, since all HTML elements are a single word.
+これは、すでに存在する、あるいは未来の HTML 要素との[コンフリクトを防止します](http://w3c.github.io/webcomponents/spec/custom/#valid-custom-element-name)。なぜなら、全ての HTML 要素は一単語だからです。
```

Comment:
`since` の訳ですが、ある程度理由・背景を理解しているとき使う理由を説明するための接続詞なので、そうすると以下な感じの訳にした方が分かりやすいと思います。

```
これは、全ての HTML 要素は 1 単語というこれまでの経緯から、既に存在する、そして将来定義される HTML 要素と衝突を防止します。
```